lib/content-api.js

```diff
@@ -60,6 +73,21 @@ function transformData(obj, dirpath) {
 }
 
 ContentApi.prototype.getRecord = function getRecord(slug) {
+  return new Promise((resolve) => {
+    this.getRecordFromApi(slug)
+      .then((record) => {
+        logger.verbose(`Loaded from content API: ${slug}`);
+        resolve(record);
+      })
+      .catch((error) => {
+        logger.error(error);
```

Comment:
Will remove the error logger for now but might use a consistent return for both file and http methods though.

@tyom what are your thoughts?